src/components/pages/UpdateList.astro

```diff
@@ -0,0 +1,138 @@
+---
+interface Props {
+  updates: Array<{
+  image: string, 
+  title: string, 
+  author?: string, 
+  tag?: string, 
+  date?: string, 
+  paragraphs: string[], 
+  ctaText: string,
+  link: string
+  }>
+}
+
+const { updates } = Astro.props;
+---
+<ul class="updates-list" id="updates-list"> 
+  {
+    updates.map( update => (
+      <li class="update">
+        <div class="update-img">
+          <img
+          src={`/img/${update.image}`}
+          alt={`Illustration for the article: "${update.title}"`}
+          width="580"
+          height="315"
+          />
+        </div>
+        <div class="update-content">
+          <h3 class="font-h4 update-title">{update.title}</h3>
+          <p class="bold update-subtitles">
+            {update.tag && (
+              <span>{update.tag}</span>
+            )}
+            {update.date && (
+              <span class="dot"></span>
+              <span>{update.date}</span>
+            )}
+            {update.author && 
+              <span class="dot"></span>
+              <span>{update.author}</span>
+            }
+          </p>
+          <div class="update-paragraphs">
+            {update.paragraphs.map( p => 
+              <p>{p}</p>
+            )}
+          </div>
+          <a
+            href={update.link}
+            class="btn secondary flexible carousel-links"
+            data-umami-event={`UpdatesList - ${update.title}`}
+            target="_blank" rel="noreferrer noopener"
+          >
+            {update.ctaText}
+          </a>
+        </div>
+      </li>
+    ))
+  }
+</ul>
+
+<style>
+  .updates-list{
+    width: 100% ;
+    display: flex;
+    flex-direction: row;
+    overflow-x: hidden;
+    padding: 0;
+  }
+
+  .update {
+    flex: 0 0 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: var(--spacing-md);
+  }
+  
+  .update-img {
+    flex-basis: 50%;     
+    min-width: 17rem;
+  }
+
+  .update-img img  {
+    height: auto;
+    object-fit: cover;
+    aspect-ratio: 580/315;
+  }
+
+  .update-content {
+    flex-basis: 50%;
+    display: flex;
+    flex-direction: column; 
+    align-items: start;
+    gap: var(--spacing-md);
+  }
+
+  .update-subtitles {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: var(--spacing-sm);
+  }
+
+  .update-subtitles .dot {
+    display: inline-block;
+    width: var(--spacing-xs); 
+    height: var(--spacing-xs);
+    background-color: var(--default-font-color);
+    border-radius: 50%;
+  }
+
+  @media screen and (max-width: 767px) {
```

Comment:
This breakpoint is too narrow so for content like this we're making it bigger
` @media (min-width: 1035px) {...}`